sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -20,6 +20,7 @@
 
 from uamqp import AMQPClient, Message, authentication, constants, errors, compat, utils
 import six
+from azure.core.utils import parse_connection_string as core_parse_connection_string
```

Comment:
can we double check the azure-core dependency version is right (>= the version we introduced this `parse_connection_string` method)?

Reply:
yep! checked and it is